manim/mobject/coordinate_systems.py

```diff
@@ -281,28 +285,27 @@ def __init__(
         make_smooth_after_applying_functions=True,
         **kwargs
     ):
-        if axis_config is None:
-            axis_config = {
-                "stroke_color": WHITE,
-                "stroke_width": 2,
-                "include_ticks": False,
-                "include_tip": False,
-                "line_to_number_buff": SMALL_BUFF,
-                "label_direction": DR,
-                "number_scale_val": 0.5,
-            }
-        self.axis_config = axis_config
-        if y_axis_config is None:
-            y_axis_config = {"label_direction": DR}
-        self.y_axis_config = y_axis_config
-
-        if background_line_style is None:
-            background_line_style = {
-                "stroke_color": BLUE_D,
-                "stroke_width": 2,
-                "stroke_opacity": 1,
-            }
-        self.background_line_style = background_line_style
+        axis_config_ = {
+            "stroke_color": WHITE,
+            "stroke_width": 2,
+            "include_ticks": False,
+            "include_tip": False,
+            "line_to_number_buff": SMALL_BUFF,
+            "label_direction": DR,
+            "number_scale_val": 0.5,
+        }
+        y_axis_config_ = {"label_direction": DR}
+        background_line_style_ = {
+            "stroke_color": BLUE_D,
+            "stroke_width": 2,
+            "stroke_opacity": 1,
+        }
+
+        self.update_default_configs(
+            (axis_config_, y_axis_config_, background_line_style_),
+            (axis_config, y_axis_config, background_line_style),
+        )
+        self.background_line_style = background_line_style_
```

Comment:
I thought it would be better to set the attributes in `Axes` and couldn't think of better names other than just adding the underscores. But underscores aren't really looking good, I'll add your suggestion.